### ISSUE TITLE
Missing functionality in PrOD

### ISSUE THREAD

**Issue by kchall** [kchall/CorpusTools#247]
It used to be that users could choose to calculate PrOD across all environments (i.e., just based on raw type or token frequency), but this option seems to be missing.

**Comment by mmcauliffe** [comment]
Yes, right, so I changed the functionality to be a bit more implicit.  If you don't specify any environments, then it does Prod based on frequency only.  I did that to reduce the amount of error messages that Prod would produce, and it aligns the dialog more with the other functions (having just two calc buttons, one for add to table, one for new table, instead of four calc buttons).  The tooltips and manual will likely need to be tweaked to reflect the new functionality.
